help/develop/how-to/test-packages.md

```diff
@@ -174,20 +175,30 @@ Progress bar on the bottom of the page shows the percentage of completed tests.
 
 ### Reviewing results
 
-Information about test results is available via tooltip or in the context panel. Selected test, category, or package to
+Information about test results is available via tooltip or in the context panel. Select test, category, or package to
 explore results. In case category/package contain multiple tests results are shown as a grid which can be added to
```

Comment:
Maybe move

> to explore results

part to the beginning of the sentence for faster navigation when you read vertically.
Or even combine this and the previous sentence.